.circleci/config.yml

```diff
@@ -659,6 +691,11 @@ workflows:
             - api-deps-check
             - api-integration-test
             - ui-unit-test
+      # Not ready
+      # - puppeteer-e2e-test-staging:
+          # filters: *filter_only_release_tags
+          # requires:
+            # - deploy-staging
```

Comment:
It is not enabled in staging workflow because the automation script that delete workspaces and billing projects are not working in staging.